packages/ember-htmlbars/lib/keywords/get.js

```diff
@@ -35,8 +35,10 @@ let DynamicKeyStream = BasicStream.extend({
 
   key() {
     let key = this.keyDep.getValue();
-    if (typeof key === 'string') {
+    if (typeof key === 'symbol') {
```

Comment:
No, if we don't support symbols already in template-land, then we don't need to support it explicitly here.